spec/features/delius_import_job_feature_spec.rb

```diff
@@ -49,23 +49,52 @@
 
     ENV['DELIUS_EMAIL_FOLDER'] = 'delius_import_feature'
     ENV['DELIUS_XLSX_PASSWORD'] = 'secret'
+  end
+
+  context "when the team is associated with an LDU" do
+    before do
+      create(:team, code: 'A')
+    end
```

Comment:
nit: can one-line short before blocks for a little easier reading
```suggestion
    before { create(:team, code: 'A') }
```